.claude/rules/frontend.md

```
@@ -0,0 +1,70 @@
+---
+paths:
+  - "components/**/*.tsx"
+  - "components/**/*.ts"
+  - "app/**/*.tsx"
+  - "app/**/*.ts"
+---
+
+# 프론트엔드 규칙
+
+## 파일 네이밍
+| 대상 | 규칙 | 예시 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)